ios/DrmModule.swift

```diff
@@ -42,7 +42,7 @@ class DrmModule: NSObject, RCTBridgeModule {
         bridge.uiManager.addUIBlock { [weak self] _, _ in
             guard
                 self?.drmConfigs[nativeId] == nil,
-                let fairplayConfig = RCTConvert.fairplayConfig(config)
+                let fairplayConfig = RCTConvert.drmConfig(config).fairplay
```

Comment:
Only use FairPlay DRM config if available as no other DRM system is supported on iOS/tvOS